app/graphql/mutations/create_encounter.rb

```diff
@@ -21,9 +21,10 @@ def resolve(input)
                               user_name: input[:user_name])
 
     if encounter.save
-      encounter_monsters = input[:encounterMonsters]
-      encounter_monsters.each do |monster_name|
-        EncounterMonster.create!(encounter_id: encounter.id, monster_name: monster_name)
+      input[:encounterMonsters].each do |monster_index|
+        EncounterMonster.create!(encounter_id: encounter.id,
+                                monster_name: monster_index.capitalize,
```

Comment:
monster names have all letters capitalized, and spaces instead of `-` `adult-black-dragon` to `Adult Black Dragon`
Instead of `.capitalize`, I think `.gsub('-', ' ').titleize` should

Reply:
Good catch! I'll update that now.